src/styles/_overrides.scss

```diff
@@ -42,3 +42,20 @@ code {
 .justify-content-flex-end {
   justify-content: flex-end !important;
 }
+
+.text-smaller {
```

Comment:
This class could maybe use a different name? To me it would be unexpected to have a class named `text-smaller` move the text with relative positioning